components/dashboard/src/projects/Project.tsx

```diff
@@ -196,6 +247,20 @@ export default function () {
                     </Item>
                 }
                 )}
+                {showAuthBanner && (
```

Comment:
**issue**: Surfacing search input and branch columns does not make sense when someone is not authorized to fetch the data, right? What do you think? Does it make sense to hide this elements like we do with the empty states in Workspaces, Projects, etc?

| BEFORE | AFTER | 
|-|-|
| <img width="1440" alt="Screenshot 2021-09-29 at 2 25 41 PM (2)" src="https://user-images.githubusercontent.com/120486/135259816-7fb1cdf3-df95-4c57-9934-166bd30e6f30.png"> | <img width="1440" alt="Screenshot 2021-09-29 at 2 25 51 PM (2)" src="https://user-images.githubusercontent.com/120486/135259821-98e56531-7a77-4a4b-8183-0c17cde0470f.png"> |

Reply:
Issue resolved